### PR TITLE
improved graphql add and update api workflow (MERGE ONLY when new api workflow goes live)

### PR DESCRIPTION
**Pull request by sundersc**
Update docs to reflect the new improved graphql add and update api workflow.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/guides/functions/cognito-trigger-lambda-dynamodb/q/platform/[platform].mdx [hunk 1]
```diff
@@ -31,17 +31,19 @@ You can skip this part, if you already have a GraphQL API with an User model.
 amplify add api
 
 ? Please select from one of the below mentioned services: GraphQL
+? Here is the GraphQL API that we will create. Select a setting to edit or continue: Name
 ? Provide API name: myapi
```
Comment (SwaySway) [inline src/pages/guides/functions/cognito-trigger-lambda-dynamodb/q/platform/[platform].mdx:35]: nit: Can we also use the same name ( `contactapi`) as mentioned in the other flows?

**Review by SwaySway — APPROVED** [review]
lgtm aside nits

**Review by david-mcafee — APPROVED** [review]
LGTM! Though I do +1 the `contactapi` callout made by @SwaySway

File: src/pages/guides/functions/cognito-trigger-lambda-dynamodb/q/platform/[platform].mdx (hunk 1 above)
Comment (sundersc) [reply src/pages/guides/functions/cognito-trigger-lambda-dynamodb/q/platform/[platform].mdx]: Updated.

**Pull request merged**